### PR TITLE
Fix returning 'expires_at' value to actual time valid for the token

### PR DESCRIPTION
**Pull request by shakedmanes**
### Current Status:
Now the `serverRegisterServe`r should work and revoke the token when its not valid anymore.

**NOTE**: I've noticed very very minor problem. When the authorization server creates the token and passes it, the time it takes is approximately `350ms`, that cause for delay in the assessment of the expiration time of the token for the `serverRegisterServer` (because the expiration time of token is only number of seconds the token is valid for, not precised time, and therefore the `serverRegisterServer` inferring by itself to add the expiration time given for token to the current time it got the token).

In conclusion, the `serverRegisterServer` got confused by thinking the token is valid with more seconds than it should be.
Luckily for us, we talking about 1~3 seconds delay, which not caused any special effect on the client side (the user simply can refresh the page or clicks many times at accordion and everything comes back to normal).

So the last step to avoid the delay is to make refactoring and optimization of the authorization server operations.

### CODE REVIEW

**Comment by shakedmanes** [comment]
Don't merge it yet.

**Comment by shakedmanes** [comment]
### Current Status:
Now the `serverRegisterServe`r should work and revoke the token when its not valid anymore.

**NOTE**: I've noticed very very minor problem. When the authorization server creates the token and passes it, the time it takes is approximately `350ms`, that cause for delay in the assessment of the expiration time of the token for the `serverRegisterServer` (because the expiration time of token is only number of seconds the token is valid for, not precised time, and therefore the `serverRegisterServer` inferring by itself to add the expiration time given for token to the current time it got the token).

In conclusion, the `serverRegisterServer` got confused by thinking the token is valid with more seconds than it should be.
Luckily for us, we talking about 1~3 seconds delay, which not caused any special effect on the client side (the user simply can refresh the page or clicks many times at accordion and everything comes back to normal).

So the last step to avoid the delay is to make refactoring and optimization of the authorization server operations.
